recipes/carveme/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "carveme" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3f58e2519af378b15b7e0d175aefa94566793503e32856d690720213a1a9d478
+
+build:
+  number: 0
+  entry_points:
+    - build_universe=carveme.cli.build_universe:main
+    - curate_universe=carveme.cli.curate_universe:main
+    - carve=carveme.cli.carve:main
+    - gapfill=carveme.cli.gapfill:main
+    - merge_community=carveme.cli.merge_community:main
+    - benchmark=carveme.cli.benchmark:main
+  script: "{{ PYTHON }} -m pip install --use-pep517 . -vv"
+  noarch: python
+  build:
+  run_exports:
+    - {{ pin_subpackage('carveme', max_pin="x.x.x") }} 
+
+requirements:
+  host:
+    - python =3.8
+    - pip
+  run:
+    - pandas =1.0
+    - python =3.8
+    - reframed =1.5
```

Comment:
please relax the pinning or remove them

Reply:
Hi! Thanks for the feedback.
I updated the recipe but it fails the tests because reframed 1.5.1 is not found. This version is just available since today on pypi, and it is necessary to relax the pinning of other packages. Is it something that just needs time, to be synchronized with pypi?